index.js

```diff
@@ -28,7 +28,13 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let moviesArray = [];
```

Comment:
We could maybe call this `movieTitlesArray` since it won't be an array of movie objects, but rather `movie` title strings.

Reply:
Yes yes we could note taken! Thank you.